src/hooks/useEVMWalletActions.ts

```diff
@@ -12,6 +12,7 @@ export function useEVMWalletActions() {
         to: calldata.to,
         data: calldata.data,
         value: calldata.value,
+        gasPrice: calldata?.gasPrice ?? undefined,
```

Comment:
Won't `calldata.gasPrice` work?

Reply:
Also is there a reason why we don't pass entire calldata (which tx object btw)?

Reply:
The `calldata?.gas Price` works. The props can be spread but we will need to resolve TS issue as it isn't exactly the same config as `sendTransaction` expects